docs/apis/streaming_guide.md

```diff
@@ -1517,11 +1517,24 @@ Stream connectors
 ----------------
 
 <!-- TODO: reintroduce flume -->
-Connectors provide an interface for accessing data from various third party sources (message queues). Currently three connectors are natively supported, namely [Apache Kafka](https://kafka.apache.org/),  [RabbitMQ](http://www.rabbitmq.com/) and the [Twitter Streaming API](https://dev.twitter.com/docs/streaming-apis).
+Connectors provide code for interfacing with various third-party systems.
+Currently these systems are supported:
 
-Typically the connector packages consist of a source and sink class (with the exception of Twitter where only a source is provided). To use these sources the user needs to pass Serialization/Deserialization schemas for the connectors for the desired types. (Or use some predefined ones)
+ * [Apache Kafka](https://kafka.apache.org/)
```

Comment:
Should we add source/sink, to show which types of connectors are im`plemented for which systems?

We can also add Java/Scala collections and file systems (HDFS) to that list.